notebooks/2_read_write.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>Is the <code>index=False</code> strictly needed? </p><p> </p><p>If we don't explain it, maybe rather not use it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I would keep it in there, as otherwise the next read_excel output has the <code>Unnamed: 0</code> column which is inconvenient. I explained the reasoning of the additional parameter.</p>